app/components/sdg/goals/show_component.html.erb

```diff
@@ -34,4 +34,8 @@
       <%= render ::Widget::Feeds::FeedComponent.new(processes_feed) %>
     </div>
   <% end %>
+
+  <section class="targets">
```

Comment:
To be consistent with the location of the CSS file (under `sdg/goals/targets.scss`), maybe this `<section class="targets">` should be inside the `sdg/goals/targets` component :thinking:.